sdk/src/azure/iot/az_iot_pnp_client_twin.c

```diff
@@ -115,16 +114,6 @@ AZ_NODISCARD az_result az_iot_pnp_client_twin_begin_property_with_status(
 
```

Comment:
Depth validation?

Reply:
Do we want to have that limitation? If someone wants to encapsulate in some other way (IE depth is maybe one greater than what we would expect)?

Reply:
Essentially we just guarantee what it looks like at "our" level and deeper. We don't guarantee anything about the surrounding stuff.